### ISSUE TITLE
Build defect 286096 com.ibm.ws.wssecurity_fat.wsscxf.1 CxfX509StrTypeTests.java - The signature or decryption was invalid (EE7 and/or EE8)

### ISSUE THREAD

**Issue by meiaus** [OpenLiberty/open-liberty#18361]
As commented in 286096

``` 
I think these 2 defects (286096, 286098) are the same issue regarding FFDC, which we'll investigate if @AllowedFFDC is required for these 2 test methods (testCxfClientSignThumbPrint, testCxfClientSignIssuerSerial)  within the same class com.ibm.ws.wssecurity.fat.cxf.x509token.CxfX509StrTypeTests.

EE7:
------Start of DE processing------ = [8/22/21, 20:20:13:973 PDT]
Exception = org.apache.ws.security.WSSecurityException
Source = org.apache.cxf.ws.security.wss4j.WSS4JInInterceptor
probeid = 395
Stack Dump = org.apache.ws.security.WSSecurityException: The signature or decryption was invalid
    at org.apache.ws.security.processor.SignatureProcessor.handleToken(SignatureProcessor.java:201)
    at org.apache.ws.security.WSSecurityEngine.processSecurityHeader(WSSecurityEngine.java:402)
    at org.apache.cxf.ws.security.wss4j.WSS4JInInterceptor.handleMessage(WSS4JInInterceptor.java:311)


EE8:

------Start of DE processing------ = [8/23/21 1:24:03:939 PDT]
Exception = org.apache.wss4j.common.ext.WSSecurityException
Source = org.apache.cxf.ws.security.wss4j.WSS4JInInterceptor
probeid = 387
Stack Dump = org.apache.wss4j.common.ext.WSSecurityException: The signature or decryption was invalid
    at org.apache.wss4j.dom.processor.SignatureProcessor.handleToken(SignatureProcessor.java:202)
    at org.apache.wss4j.dom.engine.WSSecurityEngine.processSecurityHeader(WSSecurityEngine.java:340)
    at org.apache.cxf.ws.security.wss4j.WSS4JInInterceptor.handleMessageInternal(WSS4JInInterceptor.java:332)
    at org.apache.cxf.ws.security.wss4j.WSS4JInInterceptor.handleMessage(WSS4JInInterceptor.java:213)

```

**Comment by meiaus** [comment]
To recreate the failure in local env,

1. First ran with the original test method sequence in test class (CxfX509StrTypeTests), and the test passed
```
./gradlew com.ibm.ws.wssecurity_fat.wsscxf.1:buildandrun -Dfat.test.mode=FULL -Dfat.test.class.name=com.ibm.ws.wssecu
rity.fat.cxf.x509token.CxfX509StrTypeTests -Dglobal.debug.java2.sec=false
```

testCxfClientSignThumbPrint()
testCxfClientSignIssuerSerial()
testCxfClientKeysMismatchEE7Only()
testCxfClientKeysMismatchEE8Only()

2. Then reran after moving the top test methods to the bottom in test class (CxfX509StrTypeTests), the test failed as the same in 286096.
```
./gradlew com.ibm.ws.wssecurity_fat.wsscxf.1:buildandrun -Dfat.test.mode=FULL -Dfat.test.class.name=com.ibm.ws.wssecu
rity.fat.cxf.x509token.CxfX509StrTypeTests -Dglobal.debug.java2.sec=false
```
testCxfClientKeysMismatchEE7Only()
testCxfClientKeysMismatchEE8Only()
testCxfClientSignThumbPrint()
testCxfClientSignIssuerSerial()

![image](https://user-images.githubusercontent.com/27024803/131195810-f3d23d41-f20c-4548-bcaf-546a2c982035.png)


![image](https://user-images.githubusercontent.com/27024803/131195820-49b5b50b-61e6-4c2f-9fce-71f6272b9c84.png)

**Comment by meiaus** [comment]
Fixed in #18362
